content-src/components/Base/Base.jsx

```diff
@@ -159,6 +159,9 @@ export class BaseContent extends React.PureComponent {
                 </ErrorBoundary>
               </div>
             }
+            <div className="below-search-snippet">
```

Comment:
If you want to leave this here, make sure the empty state doesn't have any extra padding. Do you need the container div?

Reply:
So... yeah. ick. I am setting a `min-height` on it to get the spacing between search and top sites right in both cases: with and without snippet. I'll try and see if I can come up with something cleaner.

Reply:
ok. was able to fix that 💃